tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -10132,6 +10132,20 @@ TEST_F(SQLiteFixture, OutputReportTabularMonthly_CurlyBraces)
         std::string colHeader = col[0];
         EXPECT_TRUE(false) << "Missing braces in monthly table for : " << colHeader;
     }
```

Comment:
We need to add a unit test for this type of thing? I think we've lost the plot on unit tests.